costumer.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict
+
+from coin import Coin
+from drink import Drink
+
+
+class Costumer:
+    def __init__(self, drink: Drink, coins: Dict[Coin, int]):
```

Comment:
Is your customer's money bound to their coin? I mean if all you care is whether they have enough money, and you'll give em a change, why bother storing their coins? Same goes for the annoying UI asking me how many coins I have.
Or is it necessary for you to keep track of the coins in the machine? If so think if it's really necessary

Reply:
It's a part of the exercise's requirements - to have the machine counting money according to specific coins. 
An integral part of the exercise is to insert the precise amount of coins from each type separately.